docs/core/diagnostics/distributed-tracing-concepts.md

```diff
@@ -101,7 +101,7 @@ or a library provided by a third-party telemetry or APM vendor. Alternately deve
 their own custom Activity telemetry collection by using
 <xref:System.Diagnostics.ActivityListener?displayProperty=nameWithType> or
 <xref:System.Diagnostics.DiagnosticListener?displayProperty=nameWithType>. ActivityListener
-supports observing any Activity regardless whether the developer has any a priori knowledge about it.
+supports observing any Activity regardless whether the developer has any a prior knowledge about it.
```

Comment:
The original "a priori" was correct, but a non-English phrase. For the new spelling to be correct, I'd remove the "a" as well:

```suggestion
supports observing any Activity regardless whether the developer has any prior knowledge about it.
```